### ISSUE TITLE
Feature Idea: Font size/checkbox/properties need basic improvments.

### ISSUE THREAD

**Issue by Cyber-Syntax** [mattermost/focalboard#4588]
## Summary

- Ability to change font size or zoom.
Importance: Low

- Ability to delete checkbox with backspace. 
Importance: Medium

- Ability to change properties with mouse, button or etc.
Importance: High

## How important this is to me and why

Importance: High/Medium/Low

Use cases:
1. Font size or zoom can be useful for who have big monitors.
2. Delete checkbox with backspace very useful for everyone. 
3. It's useful for who creating new templates for new projects.

**Comment by kamre** [comment]
@Cyber-Syntax regarding deletion of checkboxes:
 - do you mean a scenario when the text is empty for the checkbox, the input has focus and the user presses backspace?
 - should the focus go to some previous checkbox if it is available?

**Comment by Cyber-Syntax** [comment]
I totally agree with your scenarios. Also, the focus should be able to go on the previous text.
